pexpect/spawnbase.py

```diff
@@ -1,3 +1,8 @@
+try:
+    from StringIO import StringIO
```

Comment:
I think we should use `io.StringIO` in all cases, because that's strict about unicode, whereas `StringIO.StringIO` isn't.

Reply:
(By 'in all cases', I mean on Python 2 as well as Python 3 - of course we still need to use `io.BytesIO` when we're handling bytes)